thirdparty/make-many-rubies/test/all.rb

```diff
@@ -0,0 +1,8 @@
+require 'minitest/autorun'
+
+class TestPanda < MiniTest::Test
+  def test_panda_is_happy
+    panda = "sad"
+    assert_equal("happy", panda)
+  end
```

Comment:
I don't understand how this works

Reply:
I think the author is trying to demonstrate what it looks like to have failures when running `make test` - we don't actually run this test at all: it's just copied in as part of the `make-many-rubies` project

Reply:
expected fail?